src/Theme.elm

```diff
@@ -26,9 +26,11 @@ lightPurple =
 -- Emergency colours
 
 
-orange : Color
+orange : { colour : Color, string : String }
```

Comment:
I wonder about using keyword `string`

but I guess if no complaints from the compiler is OK.